Mention In and Out in README and how to empty them (#853)
JuliaLang/IJulia.jl: README.md

calls to `IJulia.clear_output(true)` followed by calls to
`display(...)` to display a new animation frame.

### Input and output history

> IJulia will store dictionarys of the user's input and output history 

Reviewer: ```suggestion
IJulia will store dictionaries of the user's input and output history 
```

for each session in exported variables called `In` and `Out`. To recall

> old inputs and outputs, simply index into `In` and `Out`. Sometimes, a user

Reviewer: ```suggestion
old inputs and outputs, simply index into them, e.g. `In[1]` or `Out[5]`. Sometimes, a user
```

may find themselves outputting large matrices or other datastructures which
will be stored in `Out` and hence not garbage collected, possibly hogging memory. 

> If you need to free this memory, empty the output dictionary:

Reviewer: ```suggestion
If you find that IJulia is using too much memory after generating large outputs, empty this output dictionary:
```
